src/types/instantsearch.ts

```diff
@@ -91,11 +92,26 @@ export type NumericRefinement = {
 
 export type Refinement = FacetRefinement | NumericRefinement;
 
+export type Router = {
```

Comment:
This should probably be an interface that is implemented by the `history` router.

Reply:
Yes – we'll be able to do this once this PR is merged.